docs/vsphere_ci/README.md

```diff
@@ -43,13 +43,14 @@ associated with this public key is what will be used by WMCO to configure VMs cr
 deploying WMCO, this private key will be provided by the user in the form of a Secret.
 
 The [autounattend.xml](scripts/autounattend.xml) file must be edited to update the value of 
-`WindowsPassword` with a user provided password. Then, it executes the following steps:
+`WindowsPassword` with a user provided password. The `ProductKey` must also be updated with a proper value.
+Then, it executes the following steps:
 
 - Runs `install-vm-tools.cmd` script which installs VMWare tools
 - Runs `configure-vm-tools.ps1` script which configures VMWare tools
 - Runs `install-openssh.ps1` script which installs and configures OpenSSH Server
-- Runs `install-docker.ps1` script which installs Docker
 - Runs `install-firewall-rules.ps1` script which configures the firewall rules
+- Runs `install-docker.ps1` script which installs Docker and restarts the machine
```

Comment:
Now that we merged containerd work already, why do we need to docker?
When we backport this PR in 5.x, we can add this line. WDYT?

Reply:
I don't think we will backport; we're treating 2022 support as a feature

Reply:
scratch that -- a 5.x means a release-4.10 backport will be needed